### ISSUE TITLE
My discoveries on the AEP format

### ISSUE THREAD

**Issue by mbasaglia** [inlife/aftereffects-project-research#6]
Just in case someone's interested, I've been trying to figure out more details on the AEP format. I've documented my discoveries in here: https://lottiefiles.github.io/lottie-docs/aep/

I focused more on AEP, but AEPX is basically the same so hopefully the information I found is useful here.

I've also written a script that export the contents of an aep file into yaml, destructuring all the binary chunks into more useful data: https://gitlab.com/mattbas/python-lottie/-/blob/master/bin/aep_dump.py

I haven't been able to figure out everything, but I managed to extract information about a lot of different layer types, shapes, keyframes, etc.

**Comment by inlife** [comment]
Oh, @mbasaglia, you are a magician! 🔥🎉

**Comment by inlife** [comment]
There was also a project for AEP parsing using golang, maybe you'll find it interesting/useful: https://github.com/boltframe/aftereffects-aep-parser

**Comment by mbasaglia** [comment]
Yeah it was useful to get started but it's very limited in its scope, I shared my findings there too

**Comment by inlife** [comment]
Added the link to the readme! 🎉
Thank you for doing this and for sharing it with everyone @mbasaglia
